test/rocketPool.js

```diff
@@ -292,7 +292,7 @@ contract('RocketPool', accounts => {
       const miniPool = RocketPoolMini.at(poolAddress);
 
       // Get the pool status
-      const poolStatus = await miniPoolFirst.getStatus.call().valueOf();
+      const poolStatus = await miniPool.getStatus.call().valueOf();
```

Comment:
Can you leave this as miniPoolFirst? There's also a miniPoolSecond thats created during the tests, so makes sense to keep the naming conventions similar.

Reply:
That is actually a correction of a typo introduced during the rewrite. If you look at the lines below and above L295 they all use `miniPool`. L295 was getting the status from the wrong pool, but it worked because `miniPool` and `miniPoolFirst` is the same thing in this case.

Reply:
Yeah I see what you've done now, looks good!